fudge-ratio/src/main/webapp/code/mainScreen.js

```diff
@@ -37,8 +37,9 @@ const showMain = () =>
     const text = get("timer-display");
     const startStop = get("start-stop");
     const finish = get("finish");
-
+    const oneDay = 1000*60*60*24;
     let id = null;
+    let keepGoing = 1;
```

Comment:
this is a boolean, right?
so:
``let keepGoing = true;``

Reply:
I was thinking the same thing as Oscar for this one.